project/templates/configs/springboard.ini

```diff
@@ -8,7 +8,9 @@ pyramid.debug_routematch = false
 
 pyramid.default_locale_name = {{default_language}}
 
-unicore.content_repo_urls = {{ucd_host}}/repos/{{repo_name}}.json
+unicore.content_repo_urls =
```

Comment:
Yeah, let's leave this out for now. We can always add it if it's introduced in IoGT